mne/preprocessing/tests/test_xdawn.py

```diff
@@ -262,4 +266,98 @@ def test_XdawnTransformer():
     pytest.raises(ValueError, xdt.inverse_transform, 42)
 
 
+def _simulate_erplike_mixed_data(n_epochs=100, n_channels=10):
+    rs = np.random.RandomState(42)
```

Comment:
we typically call these rng
```suggestion
    rng = np.random.RandomState(42)
```